justin-tan/week5/mon/game-on-rails/app/views/rps/decide.html.erb

```diff
@@ -0,0 +1,19 @@
+<h2>Welcome to the Rock Paper Scissors game!</h2>
+
+<form action="/rps/<%= params[:play] %>">
+
```

Comment:
"_In my Rock paper scissors game page, I need to input the choice twice at the select dropdown only it will prompt me to the outcome page. It won't happen for the first time and I cant seem to figure out why. Help please!_"

It's stuffing up because you have `params[:play]` in the form action. There is no play in the params yet and waits for you to submit one before it can `action '/rps/[:play]'`, which is why you need to submit it twice. 

If you just direct it to your other view page, i.e. simply type `<form action="/rps/outcome">` it will work